librz/core/cmd_quit.c

```diff
@@ -15,6 +15,41 @@ static const char *help_msg_q[] = {
 	NULL
 };
 
+RZ_IPI RzCmdStatus rz_cmd_quit_handler(RzCore *core, int argc, const char **argv) {
+	core->num->value = 0LL;
+	//exit (*input?rz_num_math (core->num, input+1):0);
+	//if (core->http_up) return false; // cancel quit when http is running
+	return RZ_CMD_STATUS_EXIT;
+}
+
+RZ_IPI RzCmdStatus rz_cmd_quit_choose_hist_proj (RzCore *core, int argc, const char **argv) {
```

Comment:
this is not right. If we go down the route of having `qyn`/`qyy`/`qnn`/`qny`, the choose_project/choose_process are not separate arguments but you have to get those from the name of the command. You can just use different handlers in each case. Please see https://github.com/rizinorg/rizin/pull/525/files#r572686967 first.

Reply:
Yes, now I have understood that. And yes, we will need to use different handlers.
👍🏾